modules/cbMap/processmap/PopupFilter.php

```diff
@@ -62,6 +62,19 @@
 			<advft_criteria_groups>[]</advft_criteria_groups>
 		</field>
 	</map>
+
+	/////////////////////////////////////////////////////////////
+	// example for getting values from inside uitype10 fields: //
+	/////////////////////////////////////////////////////////////
+	<map>
+		<modulename>Contacts</modulename>
+		<field>
+			<fieldname>account_id</fieldname>
+			<modulename>Accounts</modulename>
+			<advft_criteria>[{"groupid":1,"columnname":"contact","comparator":"e","value":"$account_id->$account_id->$phone","columncondition":""}]</advft_criteria>
```

Comment:
can we use `Accounts.phone` like we do in other parts of the application?

Reply:
ok